native/ios/README.md

```diff
@@ -32,7 +25,7 @@ a new one following this example:
 platform :ios, '12.1'
 
 target 'MeetHourSDKTest' do
-    pod 'MeetHourSDK', '~> 3.5.2'
+    pod 'MeetHourSDK', '~> 3.5.0'
```

Comment:
@chepury-bharathraj  - Documentation got removed.

Reply:
Update SDK Version to 3.5.2.